system-test/integration_test.go

```diff
@@ -34,20 +34,34 @@ import (
 )
 
 var (
-	repo   = flag.String("repo", "https://github.com/googleapis/cloud-profiler-nodejs.git", "git repo to test")
-	branch = flag.String("branch", "", "git branch to test")
-	commit = flag.String("commit", "", "git commit to test")
-	pr     = flag.Int("pr", 0, "git pull request to test")
+	repo           = flag.String("repo", "https://github.com/googleapis/cloud-profiler-nodejs.git", "git repo to test")
+	branch         = flag.String("branch", "", "git branch to test")
+	commit         = flag.String("commit", "", "git commit to test")
+	pr             = flag.Int("pr", 0, "git pull request to test")
+	runBackoffTest = flag.Bool("run_backoff_test", false, "Enables the backoff integration test. This integration test requires over 45 mins to run, so it is not run by default.")
 
 	runID             = strings.Replace(time.Now().Format("2006-01-02-15-04-05.000000-0700"), ".", "-", -1)
-	benchFinishString = "busybench finished profiling"
+	benchFinishString = "benchmark application(s) complete"
```

Comment:
Just curious, is this consistent with the bench finish string for other agents? It might be good to be consistent, for example, it'll be easier to search logs.

Reply:
Yes (or, this will be true) --I'm switching all agents to this string as I add backoff tests.